build.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-echo $1
```

Comment:
Warning for this line and others was thrown [by `shellcheck`](https://www.shellcheck.net/) about globbing/splitting.